bitnami/discourse/README.md

```diff
@@ -186,8 +187,9 @@ The following table lists the configurable parameters of the Discourse chart and
 |-------------------------------------------|---------------------------------------------------------------------------------------|--------------------------------------------------------------|
 | `postgresql.enabled`                      | Deploy PostgreSQL container(s)                                                        | `true`                                                       |
 | `postgresql.postgresqlUsername`           | PostgreSQL user to create (used by Discourse)                                         | `bn_discourse`                                               |
-| `postgresql.postgresqlPassword`           | Password for the Dicourse user                                                        | _random 10 character long alphanumeric string_               |
-| `postgresql.postgresqlPostgresPassword`   | Password for the admin user ("postgres")                                              | `bitnami`                                                    |
+| `postgresql.postgresqlPassword`           | Password for the Dicourse user - ignored if existingSecret is provided                                                       | _random 10 character long alphanumeric string_               |
+| `postgresql.postgresqlPostgresPassword`   | Password for the admin user ("postgres") - ignored if existingSecret is provided                                             | `bitnami`                                                    |
+| `postgresql.existingSecret`   | Name of an existing Kubernetes secret. The secret must have the following keys configured: `postgresql-postgres-password`, `postgresql-password`, `postgresql-replication-password`                                              | `nil`                                                    |
 | `postgresql.postgresqlDatabase`           | Name of the database to create                                                        | `bitnami_application`                                        |
 | `postgresql.persistence.enabled`          | Enable database persistence using PVC                                                 | `true`                                                       |
 | `externalDatabase.host`                   | Host of the external database                                                         | `""`                                                         |
```

Comment:
Great! Don't forget to add an entry for `externalDatabase.existingSecret` as well 😁